tests/test_mariadb.py

```diff
@@ -84,14 +90,17 @@ def _wait_for_server(connection):
 
 
 @pytest.mark.parametrize(
-    "container_per_test,db_user,db_password",
+    "container_per_test,db_user,db_password,root_pw",
     _generate_test_matrix(),
     indirect=["container_per_test"],
 )
 def test_mariadb_db_env_vars(
     container_per_test: ContainerData,
     db_user: str,
     db_password: str,
+    root_pw: str,
```

Comment:
Can be `None` too, see line 55
```suggestion
    root_pw: str | None,
```